### ISSUE TITLE
Updated API Info

### ISSUE THREAD

**Issue by MikeGruz** [hrbrmstr/newsflash#13]
It looks like the GDELT Project has provided documentation for the 2.0 API - I'm working on a project that makes use of this data, so I'm already almost finished updating the `query_tv` function to work with the changes (documentation can be found [here](https://blog.gdeltproject.org/gdelt-2-0-television-api-debuts/)).

I'll submit a pull request in the next day or so with changes. Mostly these involve names of API parameters, date formatting (they're now requiring date-time for date ranges) and the like.

**Comment by hrbrmstr** [comment]
Many thanks!

Do you know if they have they fixed some of the issues with the other API parameters? I recall there being severe issues with choosing stations and station groupings (e.g. "National" was generating errors on their back-end even in their web tools).

Def make sure you add yourself to the DESCRIPTION for this (pls). I'll get this on CRAN ASAP after the PR.

**Comment by MikeGruz** [comment]
Yep! I think the National search is working again - albeit in a wonky fashion (unlike station parameters, "National" has to be surrounded by quotes - think it has to do with sub-markets such as San Francisco being two words).

I should already be in the DESCRIPTION - added that the last go-round. Will be in touch!

**Comment by MikeGruz** [comment]
Brief update: I have emailed GDELT about their dates - for some reason JSON results for date spans under a week don't include times. Hopefully can get some resolution on that soon, otherwise short time periods really aren't worth much using the API.

**Comment by hrbrmstr** [comment]
Is it my imagination or did GDELT stop returning the text for everything but video clips?

**Comment by MikeGruz** [comment]
Yeah - it kind of looks like it. I wonder what's going on there, if it's an error or they just changed the overall operation of the API.

**Comment by seasea3** [comment]
Hey! Any update from GDELT about date spans under a week not returning times?
